exercise_data/18.csv

```diff
@@ -1,4 +1,3 @@
-#PN00-1.edf,#PN00-2.edf,#PN00-4.edf,#PN00-5.edf
```

Comment:
If you are talking about the Julia `readdlm`, it can understand them you need to give it number of rows to skip, `readdlm(...; skipstart = 1)`. DataFrames.jl provides all this automated stuff on read like the pandas one.